.github/workflows/create-release-branch.yml

```diff
@@ -45,21 +50,28 @@ jobs:
     - run: npm ci
     - run: npm install -g shelljs && npm install -g lerna
     - name: Create and Push the Release Branch
+      id: create_step
       run: |
-        echo "Creating a ${RELEASE_TYPE} release from branch ${{ github.event.ref }}"
+        echo "Creating a ${{ github.event.inputs.releaseType || 'minor' }} release from branch release/v$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"
         node scripts/create-release-branch.js
-# For future Slack Integration
-#     - name: Update Slack - Failure
-#       if: failure()
-#       run: |
-#         curl -H 'Content-Type: application/json' \
-#         -H 'Authorization: Bearer ${{ secrets.SLACK_BOT_TOKEN }}' \
-#         -X POST -d "${{ github.event.client_payload.on_failure }}" \
-#         https://slack.com/api/chat.update
-#     - name: Update Slack - Success
-#       if: success()
-#       run: |
-#         curl -H 'Content-Type: application/json' \
-#         -H 'Authorization: Bearer ${{ secrets.SLACK_BOT_TOKEN }}' \
-#         -X POST -d "${{ github.event.client_payload.on_success }}" \
-#         https://slack.com/api/chat.update
+    - id: result
+      run: echo "result=${{ job.status }}" >> $GITHUB_OUTPUT
+    - id: version
+      run: echo "version="$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"" >> $GITHUB_OUTPUT
+    - id: branch
+      run: echo "branch=release/v${{ steps.version.outputs.version }}" >> $GITHUB_OUTPUT
+    - run: echo "Release Version is ${{ steps.version.outputs.version }}"
+    - run: echo “Release Type is ${{ github.event.inputs.releaseType || 'minor' }}”
+    - run: echo “Release Branch is ${{ steps.branch.outputs.branch }}”
+
+  slack_notification:
+    if: ${{ always() }}
+    needs: create_branch
+    uses: forcedotcom/salesforcedx-vscode/.github/workflows/slack-notification.yml@cristi/slack_notify
```

Comment:
don't forget to remove your branch here

Reply:
I tried that but it wasn't working🥲 That's why I went with the full path

Reply:
well..it'll only start working after merge :) You might have to specify develop but that's the default branch I think so it should be good to go.